backends/tfhe-cuda-backend/cuda/CMakeLists.txt

```diff
@@ -62,6 +63,11 @@ else()
   set(CMAKE_CUDA_ARCHITECTURES 70)
 endif()
 
+set(CUDA_ARCH 800)
```

Comment:
We can't set cuda_arch like this, it has to be detected from the platform or set to 700 if nothing is detected. Let me look, see how we can do it.